test/routes/fileRoutesTest.js

```diff
@@ -0,0 +1,79 @@
+const chai = require('chai');
+const expect = require('chai').expect;
+const mongoose = require('mongoose');
+const fs = require('fs');
+const stream = require('stream');
+
+chai.use(require('chai-http'));
+
+const Issue = require('../../models/issueModel');
+const path = require('path');
+const File = require(path.resolve('./models/fileModel'));
+
+describe('File requests', function(){
+
+    const request = chai.request;
+
+    describe('Upload file', function(){        
+
+        describe('Valid params', function() {
+
+            it('should save file', function(done){
+                request('http://localhost:8000')
+                .post('/issues/5ab25c27f608fb1f201413e5/files')
```

Comment:
You cannot hard code id like this because each time you run tests you don't know which id Mongo will assign to new entry in collection.